### PR TITLE
Fixed clean-deploy failed

### PR DESCRIPTION
**Pull request by xuezhaojun**
Signed-off-by: xuezhaojun <zxue@redhat.com>

### CODE REVIEW

File: deploy/klusterlet/config/samples/kustomization.yaml [hunk 1]
```diff
@@ -1,3 +1,2 @@
 resources:
-  - operator_open-cluster-management_klusterlets.cr.yaml
-  - bootstrap
+  - operator_open-cluster-management_klusterlets.cr.yaml
```
Comment (qiujian16) [inline deploy/klusterlet/config/samples/kustomization.yaml]: eof

**Comment by qiujian16** [comment]
/approve

@zhiweiyin318 would you take another look?

File: deploy/klusterlet/config/samples/kustomization.yaml (hunk 1 above)
Comment (xuezhaojun) [reply deploy/klusterlet/config/samples/kustomization.yaml]: Done.

**Pull request merged**